decidim-admin/spec/system/admin_manages_organization_spec.rb

```diff
@@ -40,7 +40,15 @@
         visit decidim_admin.edit_organization_path
 
         # Makes sure in the error screenshots the editor is visible
```

Comment:
These tests have been already there in the past. I am just making the necessary modifications to these with the new editor.

These are some issues we have noticed with the editor in the past and I believe they are tested on purpose in the actual context where they live at. So that we are not re-introducing some of those bugs.

The `editor_spec` is outside of this context, it's a dummy view for testing the editor functionality. But I think it makes sense to also test it in the actual context, as we used to (particularly regarding those bugs, so that we are not re-introducing them).